docs/getting-started/terms.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Common Terms
+description: This section is a glossary of terms used within Octopus.
+position: 5
+---
+
+## Self-Hosted Octopus
+
+Installing the self-hosted Octopus Deploy server sets up the [Octopus Web Portal](/docs/getting-started/index.md#the-octopus-web-portal) and the [Octopus REST API](/docs/octopus-rest-api/index.md).
+
+The [installation documentation](/docs/installation/index.md) provides instructions for downloading, installing, and configuring your Octopus Deploy server.
+
+## Octopus Cloud
+
+[Octopus Cloud](/docs/octopus-cloud/index.md) is the hosted version of Octopus Deploy. We designed Octopus Cloud and self-hosted Octopus to provide the same functionality; however, there are some minor differences, for instance, with Octopus Cloud, we're [responsible](/docs/administration/security/index.md#responsibility) for taking backups, upgrading the service, and maintaining and monitoring the underlying systems.
+
+## The Octopus Web Portal
+
+Whether you're self-hosting the Octopus server, or using Octopus Cloud, the Octopus Web Portal is where you'll manage your infrastructure, projects, access the built-in repository, grant your team access to projects, and create your automated deployments.
+
+## Deployment Targets
+
+[Deployment Targets](/docs/infrastructure/deployment-targets/index.md)  represent the servers, machines and cloud services where your software and services will be deployed.
+
+## Environments
+
+Octopus organizes your deployment targets into groups called [Environments](/docs/infrastructure/environments/index.md) so you can promote your software through your deployment pipeline, for instance, from Development to Test and finally into Production.
+
+## Target Roles
+
+[Target roles](/docs/infrastructure/deployment-targets/target-roles/index.md) allow you to “tag” deployment targets with a specific keyword which can be used in your deployments.
+
+## Lifecycle
+
+[Lifecycles](/docs/deployment-process/lifecycles/index.md) give you control over the way releases are promoted between environments.
+
+## Package
+
+A [Package](/docs/packaging-applications/index.md) is an archive (zip, tar, NuGet) that contains all the files needed to run your software. You can host Packages in external repositories or the built-in Octopus repository. The package will need to be named correctly with a packageID, version number and format, for Octopus to recognize it. For example MyPackage.1.0.1.zip
```

Comment:
It might be worth mentioning our versioning docs here. Consider ending with "(see [versioning](/docs/packaging-applications/versioning))" after the example package zip file.